content/spin/running-apps.md

```diff
@@ -74,9 +75,46 @@ $ spin up --log-dir ~/dev/bugbash
 
 Some trigger types support additional `spin up` flags.  For example, HTTP applications can have a `--listen` flag to specify an address and port to listen on.  See the [HTTP trigger](http-trigger) and [Redis trigger](redis-trigger) pages for more details.
 
+## Monitoring Applications for Changes
+
+Spin's `watch` command rebuilds and restarts Spin applications whenever files change. You can use the `spin watch` [command](https://developer.fermyon.com/common/cli-reference#watch) in place of the `spin build` and `spin up` commands, to build, run and then keep your Spin application running without manual intervention while staying on the latest code and files.
+
+> The `watch` command accepts valid Spin [up](https://developer.fermyon.com/common/cli-reference#up) options and passes them through to `spin up` for you when running/rerunning the Spin application.
+
+By default, Spin watch monitors:
+
+* The application manifest (`spin.toml` file)
+* Any files specified in the `component.build.watch` section of the `spin.toml` file
+* Any files specified in the `component.files` section of the `spin.toml` file
+* The files specified in the `component.source` section of the `spin.toml` file
```

Comment:
Nit: Technically section should be plural because there could be multiple `component.source` sections.